content/SCALE/SCALETutorials/DataProtection/Replication/ReplicationWithEncryptionSCALE.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Setting Up a Encrypted Replication Task"
+description: "This article provides instructions on adding a replication task to a remote system and using encryption."
+weight: 40
+aliases:
+tags:
+- scalereplication
+- scalebackup
+---
+
+{{< toc >}}
+
+
+## Using Encryption in Replication Tasks
+
+TrueNAS SCALE replication allows users to create replicated snapshots of data stored in encrypted pools, datasets or zvols that on their SCALE system as a way to back up stored data to a remote system. You can use encrypted datasets in a local replication.
+
+{{< hint warning >}}
+You can set up a replication task for a dataset encrypted with a passphrase or a hex encryption key, but you must unlock the dataset before the task runs or the task fails.
+{{< /hint>}}
+
+With the implementation of rootless login and the admin user, when setting up remote replication tasks when logged in as an admin user requires selecting **Use Sudo For ZFS Commands**. 
+
+{{< include file="/content/_includes/ReplicationIntroSCALE.md" type="page" >}}
+
+Remote replication with datasets also require an SSH connection in TrueNAS. You can use an existing SSH connection if it has the same user credentials you want to use for the new replication task. 
+
+## Setting Up a Simple Replication Task Overview 
+
+This section provides a simple overview of setting up a remote replication task for an encrypted dataset. 
+It also covers the related steps you should take prior to configuring the replication task. 
+
+{{< expand "Replication Task General Overview" "v" >}}
+
+1. Set up the data storage for where you want to save replicated snapshots. 
+   
+2. Make sure the admin user has a home directory assigned. 
+   In the SCALE Bluefin early release, when creating the admin user at installation the home directory default is set to **/nonexistent**. To create an SSH connection to use in a remote replication you must assign a home directory path.
+
+   Later releases of SCALE Bluefin set the admin user home directory to one created by SCALE during the installation process, but you need to select the option to create the admin user home directory.
+
+3. Create an SSH connection between the local SCALE system and the remote system. 
+   You can do this from either **Credentials > Backup Credentials > SSH Connection** and clicking **Add** or from the **Replication Task Wizard** using the **Generate New** option in the settings for the remote system.
+
+4. Unlock the encrypted dataset(s) and export the encryption key to a text editor like Notepad.
+
+5. Go to **Data Protection > Replication Tasks** and click **Add** to open the **Replication Task Wizard**. 
+   You then specify the from and to sources, task name, and set the schedule.
+  
+   Setting options change based on the source selections. Replicating to or from a local source does not requires an SSH connection.
```

Comment:
Suggest reword: 'Replicating to or from a local source..." to "A to or from source that is local, does not..." the first one almost sounds like any time you are replicating to or from a local source (which is always) you never need an SSH connection. Or perhaps "When a source (to or from) is remote, an SSH Connection is required"